src/components/Boundary.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import * as Sentry from '@sentry/browser';
+
+class Boundary extends Component {
+    constructor(props) {
+        super(props);
+        this.state = { eventId: null };
+    }
```

Comment:
```suggestion
    state = { eventId: null };
```
`state` можно объявлять свойством. Не обязательно для него переопределять конструктор